.pre-commit-config.yaml

```diff
@@ -66,6 +52,27 @@ repos:
     hooks:
     -   id: black
         args: [--config, pyproject.toml]
+        exclude: |
+            (?x)^(
+                nilearn/datasets/tests/data/localizer_behavioural.tsv
+                | nilearn/externals/.*
+                | nilearn/plotting/data/js/.*js
+                | nilearn/_version.py
+                | examples/01_plotting/plot_3d_map_to_surface_projection.py
+                | examples/01_plotting/plot_surf_atlas.py
```

Comment:
I was just wondering why we don't want to blackify all these files.

Reply:
Long story short, using black was implemented "module by module" and the files left there are those that are still involved in some pull requests.

Several of those can be "blackified" once we have made progress on the surface object front.